librz/util/calc.c

```diff
@@ -257,6 +257,7 @@ static RzNumCalcValue prim(RzNum *num, RzNumCalc *nc, int get) {
 		} else {
 			error(num, nc, " ')' expected");
 		}
+		return v;
```

Comment:
love this catch

Reply:
It doesn't really change anything in terms of behaviour, but I think it is a nice thing to be aware of these things going forward. I've tried to be conservative in this initial pass.